Blasted.io/Bullet.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <SFML/Graphics.hpp>
+#include "Colors.h"
+class Bullet
+{
+public:
+	sf::Vector2f bulletSpeed;
+	sf::CircleShape bulletShape;
+
+	float bulletSpeedIncrease;
+
+	sf::Clock clock;
+	float delay = 0.7f;
```

Comment:
```suggestion
	float delay;
```
Delay is initialized in constructor